bot/exts/blurple_formatter.py

```diff
@@ -2,11 +2,14 @@
 import traceback
 
 import discord
+from botcore.utils.logging import get_logger
 from botcore.utils.regex import FORMATTED_CODE_REGEX
 from discord.ext import commands
 
 from bot.bot import SirRobin
-from bot.utils import blurple_formatter
+from bot.utils import blurple_formatter, services
+
+log = get_logger(__name__)
```

Comment:
This doesn't seem to be used anywhere

Reply:
Ah, yes you're are right. This was left from another attempt I made - oops. Fixed in e345ae5d64afbf7f93a6dc637665009c9451568d